### PR TITLE
pacific: mgr/stats: change in structure of perf_stats o/p

### PR DESCRIPTION
**Pull request by neesingh-rh**
backport tracker: https://tracker.ceph.com/issues/57279, https://tracker.ceph.com/issues/57277 and https://tracker.ceph.com/issues/57331

---

backport of https://github.com/ceph/ceph/pull/46068
parent tracker: https://tracker.ceph.com/issues/56162, https://tracker.ceph.com/issues/56169 and https://tracker.ceph.com/issues/55861

this backport was staged using ceph-backport.sh version 16.0.0.6848
find the latest version at https://github.com/ceph/ceph/blob/master/src/script/ceph-backport.sh

### CODE REVIEW

**Review by anthonyeleven — APPROVED** [review]
Docs Lgtm

**Review by joscollin — CHANGES_REQUESTED** [review]

File: qa/tasks/cephfs/test_mds_metrics.py [hunk 1]
```diff
@@ -498,15 +517,92 @@ def test_perf_stats_stale_metrics(self):
...
+            if not (client_metadata_a[i]['hostname']):
+                raise RuntimeError("hostname of fs1 not found!")
+            if not (client_metadata_a[i]['valid_metrics']):
+                raise RuntimeError("valid_metrics of fs1 not found!")
+
+        for i in client_metadata_b:
+            if not (client_metadata_b[i]['hostname']):
+                raise RuntimeError("hostname of fs2 not found!")
+            if not (client_metadata_b[i]['valid_metrics']):
+                raise RuntimeError("valid_metrics of fs2 not found!")
```
[the remark this answers is not in the document]
Comment (neesingh-rh) [reply qa/tasks/cephfs/test_mds_metrics.py:547]: Fixed!! I guess it came during resolving conflict.My bad.

[the remark this answers is not in the document]
Comment (joscollin) [reply qa/tasks/cephfs/test_mds_metrics.py:547]: You should add a `Conflicts:` section.  

Please see:
https://tracker.ceph.com/projects/ceph-releases/wiki/HOWTO_backport_commits_old
https://github.com/ceph/ceph/blob/main/SubmittingPatches-backports.rst#cherry-picking-rules

I guess the doc commit also had conflicts?

[the remark this answers is not in the document]
Comment (neesingh-rh) [reply qa/tasks/cephfs/test_mds_metrics.py:547]: Updated the commits!

**Comment by joscollin** [comment]
I'm closing this one. A new backport is created which doesn't have commit edits: https://github.com/ceph/ceph/pull/47851

**Pull request closed without merging**